<launch>
  <arg name="role_name" default="ego_vehicle" />
  <arg name="local_length" default="50.0" />
  <arg name="spawn_point" default='229.5,-86,2.5,0,0,-90'/>

  <node pkg="auto_drive" type="plan" name="plan" output="screen">
    <param name="role_name" type="string" value="$(arg role_name)" />
    <param name="local_length" type="double" value="$(arg local_length)" />
    <rosparam command="load" file="$(find auto_drive)/config/reference_line.yaml" />
    <rosparam command="load" file="$(find auto_drive)/config/planner_config.yaml" /> 
  </node>

  <include file="$(find carla_waypoint_publisher)/launch/carla_waypoint_publisher.launch" />
  <node name="rviz" pkg="rviz" type="rviz" required="True" args="-d $(find auto_drive)/rviz/plan.rviz" />
</launch>